### DISCUSSION TITLE
MTEX Workshop 2025

### DISCUSSION THREAD

**Discussion by ralfHielscher** [mtex-toolbox/mtex#2249] [Announcements]
Dear MTEX community,

we are very happy to announce the MTEX workshop 2025 in Freiberg. We had such a productive and relaxed time at the last workshop!  We can't wait to see what you have been using MTEX for over the past year and to show you the new features of MTEX 6.0. As usual the first week will be an introductory course to texture analysis and MTEX. Once you've have attended, you will be certified for 5 ETCS points. The second week will be a get together of MTEX users to share their experiences and present their latest projects. We have also added a day to the second week to make space for a nice excursion. 

More information can be found at the workshop page https://mtex-toolbox.github.io/workshop25 which will be constantly updated.

Participation will be possible in person and online. However, everybody who attended the workshop in person will tell you - it was worth the additional effort 😄

All the best,
The MTEX team.

![Gruppenfoto MTEX 2024](https://github.com/user-attachments/assets/5a9f71c3-5013-4757-bb86-280cb3a847d4)
![20240306_113809](https://github.com/user-attachments/assets/7490890b-bf76-47ae-8030-6201db339b74)
![20240306_083749](https://github.com/user-attachments/assets/02da8631-c2b3-4803-9019-9ab102160e15)

**Comment by gmcastroguiza** [comment]
Dear @ralfHielscher ,
I have been working for some time in an algorithm to quantify prior austenite grain boundaries, and I'd be interested in sharing my work during the workshop. Is the workshop still open for presentations? If so, how can I submit my proposal and which is the deadline for such a submission?
Kind regards,
Gabriel Castro

**Comment by ralfHielscher** [comment]
Dear Gabriel,
it is wonderful that you want to share your work at the upcoming MTEX workshop. All you have to do is to register at https://tickets.acatrain.net/mtex-2025/. 
You correctly observed that there seems to be no way to specify a title for a talk. This needs to be changed. In the meantime it 
is easiest to send an email to Ralf.Hielscher@math.tu-freiberg.de.
I looking forward to your talk.

All the best, Ralf.
